[2024-06-10T14:09:03.911+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-10T14:09:03.931+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_toll_data.transform_data manual__2024-06-10T14:08:28.852914+00:00 [queued]>
[2024-06-10T14:09:03.941+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_toll_data.transform_data manual__2024-06-10T14:08:28.852914+00:00 [queued]>
[2024-06-10T14:09:03.942+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-10T14:09:03.955+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transform_data> on 2024-06-10 14:08:28.852914+00:00
[2024-06-10T14:09:03.963+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=92) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-10T14:09:03.965+0000] {standard_task_runner.py:63} INFO - Started process 94 to run task
[2024-06-10T14:09:03.966+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'ETL_toll_data', 'transform_data', 'manual__2024-06-10T14:08:28.852914+00:00', '--job-id', '42', '--raw', '--subdir', 'DAGS_FOLDER/ETL_toll_data.py', '--cfg-path', '/tmp/tmp99f9sekg']
[2024-06-10T14:09:03.967+0000] {standard_task_runner.py:91} INFO - Job 42: Subtask transform_data
[2024-06-10T14:09:04.016+0000] {task_command.py:426} INFO - Running <TaskInstance: ETL_toll_data.transform_data manual__2024-06-10T14:08:28.852914+00:00 [running]> on host 2c4b4cb333a5
[2024-06-10T14:09:04.196+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='21521333@gm.uit.edu.vn' AIRFLOW_CTX_DAG_OWNER='quan' AIRFLOW_CTX_DAG_ID='ETL_toll_data' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2024-06-10T14:08:28.852914+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-10T14:08:28.852914+00:00'
[2024-06-10T14:09:04.197+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-10T14:09:04.243+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-10T14:09:04.244+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-10T14:09:04.252+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ETL_toll_data, task_id=transform_data, run_id=manual__2024-06-10T14:08:28.852914+00:00, execution_date=20240610T140828, start_date=20240610T140903, end_date=20240610T140904
[2024-06-10T14:09:04.301+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-10T14:09:04.329+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-10T14:09:04.333+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
